docs/source/cli-options.md

```diff
@@ -69,65 +69,83 @@ Windows installers have the following CLI options available:
 - `/RegisterPython=[0|1]`: Whether to register Python as default in the Windows registry. Defaults
   to `1`. This is preferred to `AddToPath`.
 - `/Q` (quiet): do not report to the console in headless mode. Only relevant when used with `/S`
-  (see below).
+  (see below). Also works for the uninstallers.
 
 You can also supply [standard NSIS flags](https://nsis.sourceforge.io/Docs/Chapter3.html#installerusage), but only _after_ the ones mentioned above:
 
 - `/NCRC`: disables the CRC check.
 - `/S` (silent): runs the installer or uninstaller in headless mode. Installers created with
-  `constructor 3.10` or later will report information to the active console.
+  `constructor 3.10` or later will report information to the active console. Note that while the
+  installer output will be reported in the active console, the uninstaller output will happen in
+  a new console. See below for different invocation examples.
 - `/D` (directory): sets the default installation directory. Note that even if the path contains
   spaces, it must be the last parameter used in the command line and must not contain any quotes.
   Only absolute paths are supported. The uninstaller uses `_?` instead of `/D`.
 
-### Examples:
-
-> Note that the NSIS installers will not write any output to the terminal. You will not see any
-> output, even if the installation fails. You can check the Task Manager to see if the installer is
-> running, or poll the installation directory to see if the installation has finished.
+### Examples
 
-Run the installer in silent mode:
+Run the installer in headless mode:
 
-```batch
-> cmd.exe /c start /wait my_installer.exe /S
+`````{tab-set}
+````{tab-item} CMD
+```pwsh
```

Comment:
```suggestion
```batch
```

Reply:
I used pwsh on purpose here because the batch syntax highlighting didn't shown any colours at all.